packages/components/src/Enumeration/Enumeration.component.js

```diff
@@ -48,26 +69,35 @@ function Enumeration({ displayMode, headerDefault, headerInput,
 	return (
 		<div className={enumerationClasses()}>
 			{getHeader()}
-			<Items items={items} itemsProp={itemsProp} currentEdit={currentEdit} />
+			<Items
+				items={items}
```

Comment:
I think "selectedItems" is redundant with "items", it would be better to that each item of "items" contains the "selected" state. What do you think?